README.md

```diff
@@ -94,6 +94,7 @@ in February 2016.
 |[Düsseldorf][duesseldorf-wikipedia]|[City of Düsseldorf][duesseldorf-markets]|
 |[Erlangen][erlangen-wikipedia]|[City of Erlangen][erlangen-markets]|
 |[Essen][essen-wikipedia]|[EVB][essen-markets]|
+|[Frankfurt am Main][frankfurt-wikipedia]|[City of Frankfurt am Main][frankfurtmain-markets]|
```

Comment:
The markdown link does not match the bottom link. You can always check the [preview of the `README.md` in you local branch](https://github.com/masterofpuppets103/wo-ist-markt.github.io/tree/market-data-frankfurtmain).

![readme](https://cloud.githubusercontent.com/assets/144518/24218906/4ee63f8a-0f45-11e7-9dbb-2c68c3d473cf.png)